eva_submission/evapro/find_from_ena.py

```diff
@@ -0,0 +1,191 @@
+import re
```

Comment:
Do you think there's anything in any of this code that we (or ENA) might not want to make public? Just mentioning as the previous script was in a private repo...

Reply:
The internal of the ENA database are not for say public but we're not really exposing all of it. I think it should be ok.